content/influxdb/cloud/write-data/troubleshoot.md

```diff
@@ -74,13 +77,62 @@ If you notice data is missing in your bucket, do the following:
 - Verify the timestamps match the [precision parameter](/influxdb/cloud/write-data/#timestamp-precision).
 - Minimize payload size and network errors by [optimizing writes](/influxdb/cloud/write-data/best-practices/optimize-writes/)
 
-### Troubleshoot rejected points
-
-InfluxDB may reject points even if the HTTP request returns "Success".
-If some of your data did not write to the bucket, check for [field type](/influxdb/cloud/reference/key-concepts/data-elements/#field-value) differences between the missing data point and other points that have the same [series](/influxdb/cloud/reference/key-concepts/data-elements/#series).
-For example, did you attempt to write `string` data to an `int` field?
-
-InfluxDB rejects points for the following reasons:
-- The **batch** contains another point with the same series, but one of the fields has a different value type.
-- The **bucket** contains a saved point with the same series, but one of the fields has a different value type.
-
+### Review rejected points
+
+InfluxDB may have rejected points even if the HTTP request returned "Success".
+To get a log of rejected data points, query the [`rejected_points` measurement](/influxdb/cloud/reference/internals/system-buckets/#_monitoring-bucket-schema) in your organization's `_monitoring` bucket.
+
+```js
+from(bucket: "_monitoring")
+  |> range(start:-1h)
+  |> filter(fn:(r) =>
+    r._measurement == "rejected_points"
+  )
+```
+
+InfluxDB returns `rejected_points` log entries.
+Each entry contains the `bucket` where the rejection occurred and a `reason` for the rejection.
+
+```sh
+rejected_points,bucket=YOUR_BUCKET,field=YOUR_FIELD,gotType=Float,
+  measurement=YOUR_MEASUREMENT,reason=type\ conflict\ in\ batch\ write,
+  wantType=Integer count=1i 1627906197091972750
+```
+
+#### Troubleshoot field type conflicts
+
+Field type conflicts occur if you attempt to write different data types to a field.
+For example, you attempt to write `string` data to an `int` field.
+
+{{% note %}}
+If you want to reject batches that contain points with field type conflicts, use an [explicit bucket schema](/influxdb/cloud/organizations/buckets/bucket-schema/). Explicit bucket schemas reject the write request and return an HTTP `400 Bad Request` status code and a JSON response body like the following:
+```json
+{
+  "code": "invalid",
+  "message": "partial write error (2 written): unable to parse 'air_sensor,service=S1,sensor=L1 temperature=\"90.5\",humidity=70.0 1632850122': schema: field type for field \"temperature\" not permitted by schema; got String but expected Float"
```

Comment:
That message will shortly change to something more like this:

```
partial write error (2 accepted): 3/5 points rejected; first error: schema: field type for field \"temperature\" not permitted by schema; got String but expected Float
```

because we'll won't be stopping at the first point that fails schema validation.

Reply:
Created new issue.